src/vma/dev/qp_mgr.h

```diff
@@ -215,21 +216,28 @@ friend class cq_mgr_mlx5;
 	uint16_t            m_n_ip_id_offset;
 
 	mgid_ref_count_map_t  m_attach_mc_grp_ref_cnt;
+	bool                m_skip_tx_release;
```

Comment:
Unnecessary - please remove this. If there are no tx buffers to release `qp_mgr::release_tx_buffers()` will return without doing anything.